src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/Utils.java

```diff
@@ -1093,28 +1026,23 @@ public TypeElement getFirstVisibleSuperClassAsTypeElement(TypeElement te) {
      *          be found.
```

Comment:
Although not edited, this comment could be improved by defining the term "closest visible".
In particular, I notice the implementation includes reference to `@hidden` tags.